ext/standard/math.c

```diff
@@ -126,6 +126,24 @@ static inline double php_round_helper(double value, int mode) {
 
 			return integral;
 
+		case PHP_ROUND_CEILING:
+			if (value > 0.0) {
+				return fractional == 0 ? integral : integral + 1.0;
+			}
+			return integral;
+
+		case PHP_ROUND_FLOOR:
+			if (value < 0.0) {
+				return fractional == 0 ? integral : integral - 1.0;
+			}
+			return integral;
+
+		case PHP_ROUND_TOWARD_ZERO:
+			return integral;
+
+		case PHP_ROUND_AWAY_FROM_ZERO:
+			return fractional == 0 ? integral : integral + copysign(1.0, integral);
```

Comment:
```suggestion
			if (fractional > 0.0) {
				return integral + copysign(1.0, integral);
			}

			return integral;
```

For visual consistency with the other cases.